microsoft/testsuites/core/provisioning.py

```diff
@@ -277,7 +277,7 @@ def verify_stop_start_in_platform(
         The reboot times is summarized after the test is run
         """,
         priority=3,
-        timeout=7200,
+        timeout=9999,
```

Comment:
Did you test it? For the 7200, how many reboots can be performed when it times out?

Reply:
it was between 90-97 for certain sizes.

Reply:
Ok, please use 3*3600 which means 3 hours instead of a magic number 9999.

Reply:
Used 10800 s instead of 9999.